ethereum/remix-project
# Parse circom code and display errors and warnings in the editor

Fixes https://github.com/ethereum/remix-project/issues/3854

#### To Test
- Add a new file to the file-explorer `circuits/example.circom`
- Write a simple circuit that contains an error
```
pragma circom 2.0.0;

/*This circuit template checks that c is the multiplication of a and b.*/  

template Multiplier2 () {  

   // Declaration of signals.  
   signal input a;  
   signal input b;  
   signal output c;  

   // Constraints.  
   c <== a * b;;
}

```
- You should see a red squiggly line in the editor, indicating that there is an error in the syntax of the code.
